web/test/pages/profile/profile-starting.test.tsx

```diff
@@ -948,34 +947,10 @@ describe("profile - starting business", () => {
     expect(screen.getByTestId("snackbar-alert-ERROR")).toBeInTheDocument();
   });
 
-  describe("business name required", () => {
-    const required: OperatingPhaseId[] = ["FORMED_AND_REGISTERED", "UP_AND_RUNNING"];
-    const notRequired: OperatingPhaseId[] = ["GUEST_MODE", "NEEDS_TO_FORM", "NEEDS_TO_REGISTER_FOR_TAXES"];
-
-    for (const phase of required) {
-      it(`prevents user from saving if business name in ${phase} phase`, async () => {
-        const business = generateBusinessForProfile({
-          profileData: generateProfileData({
-            businessPersona: "STARTING",
-            operatingPhase: phase,
-            businessName: "",
-            legalStructureId: "limited-liability-company",
-          }),
-        });
-        renderPage({ business });
-        fireEvent.blur(screen.getByLabelText("Business name") as HTMLElement);
-
-        clickSave();
-        await waitFor(() => {
-          expect(
-            screen.getByText(Config.profileDefaults.fields.businessName.default.errorTextRequired)
```

Comment:
This really has no bearing on functionality at all, just kindof a house keeping thought. Given we don't need this error message any more, should we delete `businessName.default.errorTextRequired` from `profile.json`?

Reply:
Great catch!

Reply:
In the future you don't need to include "This really has no bearing on functionality at all, just kindof a house keeping thought."

That is a completely reasonable review comment and it _should_ block merging of this story.